src/components/NewDraftWizard/WizardForms/WizardDOIForm.tsx

```diff
@@ -15,38 +15,45 @@ import { resetCurrentObject } from "features/wizardCurrentObjectSlice"
 import { addDoiInfoToFolder } from "features/wizardSubmissionFolderSlice"
 import { useAppSelector, useAppDispatch } from "hooks"
 import schemaAPIService from "services/schemaAPI"
+import { DoiFormDetails, FormObject } from "types"
 import { dereferenceSchema } from "utils/JSONSchemaUtils"
 
 const useStyles = makeStyles(theme => ({
   form: { ...theme.form },
 }))
 
-const DOIForm = ({ formId }: { formId: string }): any => {
+const DOIForm = ({ formId }: { formId: string }) => {
   const [dataciteSchema, setDataciteSchema] = useState({})
   const locale = useAppSelector(state => state.locale)
 
   useEffect(() => {
     let isMounted = true
     const getDataciteSchema = async () => {
       let dataciteSchema = sessionStorage.getItem(`cached_datacite_schema`)
+      let parsedDataciteSchema: FormObject | undefined = undefined
+
       if (!dataciteSchema || !new Ajv().validateSchema(JSON.parse(dataciteSchema))) {
         try {
           const response = await schemaAPIService.getSchemaByObjectType("datacite")
-          dataciteSchema = response.data
+          dataciteSchema = JSON.stringify(response.data)
           sessionStorage.setItem(`cached_datacite_schema`, JSON.stringify(dataciteSchema))
```

Comment:
We have `JSON.stringify` twice

Reply:
Latest changes has only one on variable set.